geth/jail/jail.go

```diff
@@ -204,8 +205,10 @@ func (jail *Jail) Send(call otto.FunctionCall) (response otto.Value) {
 		var resErr error
 		var res *otto.Object
 
-		switch req.Method {
-		case params.SendTransactionMethodName:
+		switch {
```

Comment:
This solution might not scale as we would need to create new methods like `ExecuteXXX()` with every exception. It looks like we need:
* `policy.ExecuteSendTransaction()` as it's unique, 
* `policy.ExecuteLocally()` as some commands can only be executed locally,
* `policy.Execute???()` as some can be executed using both clients.